hubmap/frontend/src/commons/human-svg-utils.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-plusplus */
+
+export function updateText(tspanId, txt) {
+  const spanEl = document.getElementById(tspanId);
+  while (spanEl.firstChild) {
+    spanEl.removeChild(spanEl.firstChild);
+  }
+  spanEl.appendChild(document.createTextNode(txt));
+}
+
+export function showToolTip(evt) {
+  const humanSvg = document.getElementById('human');
+  const tooltip = humanSvg.getElementById('tooltip');
+  const tooltipText = tooltip.getElementsByTagName('text')[0];
+  const tooltipRects = tooltip.getElementsByTagName('rect');
+  const CTM = humanSvg.getScreenCTM();
+  const mousex = (evt.clientX - CTM.e + 6) / CTM.a;
+  const mousey = (evt.clientY - CTM.f + 20) / CTM.d;
+  tooltip.setAttributeNS(null, 'transform', `translate(${mousex} ${mousey})`);
+  tooltip.setAttributeNS(null, 'visibility', 'visible');
+  const lines = evt.target.getAttributeNS(null, 'data-tooltip-text').split(',');
+  // eslint-disable-next-line prefer-destructuring
+  tooltipText.firstChild.data = lines[0];
+  updateText('tspanid', lines[1]);
+  const length = tooltipText.getComputedTextLength();
+  for (let i = 0; i < tooltipRects.length; i++) {
+    tooltipRects[i].setAttributeNS(null, 'width', length + 10);
+    tooltipRects[i].setAttributeNS(null, 'height', 30);
+    tooltipRects[i].setAttributeNS(null, 'font-size', 2);
```

Comment:
Having a util function to set attributes from an Object would make this more readable. Something like this on SO: https://stackoverflow.com/questions/12274748/setting-multiple-attributes-for-an-element-at-once-with-javascript 
```
function setAttributes(el, attrs) {
  for(var key in attrs) {
    el.setAttribute(key, attrs[key]);
  }
}
```